### PR TITLE
Feature/nex 243/override publish controllers

### PR DESCRIPTION
**Pull request by ferenckiss89**
Jira:
- https://oat-sa.atlassian.net/browse/NEX-243
- https://oat-sa.atlassian.net/browse/NEX-244

**NEX-243 (Deliveries page: 3-way form switcher)**
- [x] Requires https://github.com/oat-sa/extension-tao-delivery-rdf/pull/273
- [x] Requires https://github.com/oat-sa/extension-tao-lti-consumer/pull/16
By relying on exportable Frontend modules from `taoDeliveryRdf` and `taoLtiConsumer`, we can keep the logic in the controller here to a minimum.

The Backend part is **not** production-ready, it was created as a rough mock to put the form fields on the page which the FE then wires up. So, it cannot yet process forms submitted to the controller. (Also, the whole architecture might be wrong...)

To ease development, the `delivery-new` action button defined in `controller/structures.xml` has been assigned a temporary id and name, different from the one in `taoDeliveryRdf` which it will eventually override.

```
taoDeliveryRdf / wizard.js
[reusable: deliveryFormHelper, testProvider.listTests]
  |
  taoLtiConsumer / wizard.js
  [reusable: deliveryFormHelper, ltiProvider.listLtiProviders]
    |
    taoDeliverConnect / deliveries.js
```

**NEX-244 (Tests page: 2-way switcher)**
- [ ] Requires https://github.com/oat-sa/tao-core-ui-fe/pull/47
- The BE controller is very simple
- Frontend uses the tabs component
- Providers dropdown is still hardcoded in template

### CODE REVIEW

File: views/js/controller/Publish/tests.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    'css!taoDeliverConnectCss/tests.css'
+], function (__, urlHelper, actionManager, resourceProviderFactory, destinationSelectorFactory, feedback, taskQueue, tabsComponent) {
+    'use strict';
+
+    return {
+       
+        start : function start() {
+
+            // Selectors
+            const $container = $('.selector-container');
```
Comment (oatymart) [inline views/js/controller/Publish/tests.js]: jquery needs to be added to the dependencies.

File: views/scss/tests.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.test-multi-publishing {
```
Comment (oatymart) [inline views/scss/tests.scss]: Can it all be scoped to a block?

File: views/js/controller/Publish/tests.js (hunk 1 above)
Comment (ferenckiss89) [reply views/js/controller/Publish/tests.js]: true, fixed.

File: views/scss/tests.scss (hunk 2 above)
Comment (ferenckiss89) [reply views/scss/tests.scss]: yes

**Pull request merged**